CoreScriptsRoot/Modules/Settings/Pages/Help.lua

```diff
@@ -162,23 +162,17 @@ local function Initialize()
     accessoriesFrame.Parent = parentFrame
 
     local miscFrame = nil
-    local hideHudSuccess, hideHudFlagValue = pcall(function() return settings():GetFFlag("AllowHideHudShortcut") end)
```

Comment:
why are we removing this? IIRC sometimes marketing team uses this flag to take in-game footage w/o any UI